docker-compose-dev.yml

```diff
@@ -25,11 +27,11 @@ services:
       - REQUESTS_CA_BUNDLE=$CACERT_LOCATION
     image: basegun-backend:${TAG:-2.0}-dev
     ports:
-      - 5000:5000
+      - 8000:8000
     volumes:
-      - $PWD/backend/src:/app/src
-      - $PWD/backend/tests:/app/tests
-      - $PWD/backend/logs:/app/logs
+      - ./backend/src:/app/src

```

Comment:
I heard that for volumes Docker doesn't like much relative paths, why not leave $PWD ?